cmd/discord-provider/server/operations.go

```diff
@@ -16,8 +21,41 @@ func NewServer() *Server {
 }
 
 func (server *Server) SendMessage(ctx context.Context, message *discord_provider.SendMessageRequest) (*discord_provider.SendMessageResponse, error) {
-	// Sends a message to a Discord webhook here
+	url, found := os.LookupEnv("DISCORD_WEBHOOK_URL")
+	if !found {
+		return &discord_provider.SendMessageResponse{}, errors.New("the env var DISCORD_WEBHOOK_URL is not set to a value")
+	}
+
+	discordMessage := discordMessage{
+		Username: "calidum-rotae",
+		Embeddeds: []embedded{
+			{
+				Title: "New submission",
+				Description: discordSenderInformation{
+					firstName:      message.Sender.FirstName,
+					lastName:       message.Sender.LastName,
+					email:          message.Sender.Email,
+					phoneNumber:    message.Sender.PhoneNumber,
+					requestDetails: message.RequestDetails,
+					requestService: message.RequestService,
+				}.String(),
+				Color: "16745728", // orange
+				Footer: footer{
+					Text: "By calidum-rotae services",
+				},
+			}},
+	}
+
+	payload := new(bytes.Buffer)
+	err := json.NewEncoder(payload).Encode(discordMessage)
+	if err != nil {
+		return &discord_provider.SendMessageResponse{}, errors.New(fmt.Sprintf("failed to encode the message %v", discordMessage))
```

Comment:
```suggestion
		return &discord_provider.SendMessageResponse{}, fmt.Errorf("failed to encode the message %v", discordMessage)
```
`fmt.Errorf(...)` removes the need to call both functions, best of both worlds 😄